src/views/ManageList.jsx

```diff
@@ -1,57 +1,13 @@
 import { useState } from 'react';
-import { addItem } from '../api';
 import { shareList } from '../api/firebase.js';
 
-export function ManageList({ listPath, userId, data }) {
-	const initialItemFormState = {
-		itemName: '',
-		daysUntilNextPurchase: 0,
-	};
-	const [newItem, setNewItem] = useState(initialItemFormState);
+export function ManageList({ listPath, userId }) {
```

Comment:
Reference other issue: I will be adding a list deleting functionality to this component in issue #40, but it is neat to see how much code was moved out of this as you refactored for this task!